Content.Server/Prayer/PrayerSystem.cs

```diff
@@ -104,5 +111,13 @@ public void Pray(ICommonSession sender, PrayableComponent comp, string message)
 
         _chatManager.SendAdminAnnouncement($"{Loc.GetString(comp.NotificationPrefix)} <{sender.Name}>: {message}");
         _adminLogger.Add(LogType.AdminMessage, LogImpact.Low, $"{ToPrettyString(sender.AttachedEntity.Value):player} sent prayer ({Loc.GetString(comp.NotificationPrefix)}): {message}");
+		
+        /// Admin sound
+        if (_gameTiming.CurTime > comp.NextRing) {
+            _audioSystem.PlayGlobal(comp.SoundForGod,
+                Filter.Empty().AddPlayers(_adminManager.ActiveAdmins), false, AudioParams.Default.WithVolume(comp.VolumeForGod));
+            comp.NextRing = _gameTiming.CurTime + comp.RingPeriod;
```

Comment:
Use AudioParams for volume and just overwrite the stream with stop + play.

Reply:
I could not figure this out. But in my testing it also did not work the way I was doing it, so I just decided to nuke the feature idea and pick .oggs that don't need adjusting.

Reply:
you can setup params in yml:

```yml
soundForGod:
  path: /Audio/Effects/voicemail.ogg
  params:
    pitch: ?
    variation: ?
    volume: ?
```